### ISSUE TITLE
Type of station.guard_mode changes to number when changed manually, turns back to string when fetched from api

### ISSUE THREAD

**Issue by MaxWinterstein** [bropat/ioBroker.eufy-security#40]
**Describe the bug**  
While playing around I found some strange issues with my visualization. The type of `station.guard_mode` seems to be incosistent.

**To Reproduce**  
1. Fire up some command at the iobroker host
2. fetch current state of the station.guard_mode
```bash
root@iobroker:/opt/iobroker# iobroker state get eufy-security.0.T8010NXXXXXXX.station.guard_mode                                       
{"val":"1","ack":true,"ts":1609259693908,"q":0,"from":"system.adapter.eufy-security.0","lc":1609259693908}    
```
Note the `val` is quoted.

3. Change the value from e.g. the object overview to something else, e.g. from current `Home` (`1`) to `away` (`0`)
4. quickly run the command to fetch the current state again:
```bash
root@iobroker:/opt/iobroker# iobroker state get eufy-security.0.T8010NXXXXXXX.station.guard_mode                                       
{"val":0,"ack":true,"ts":1609259947464,"q":0,"from":"system.adapter.admin.0","user":"system.user.admin","lc":1609259947464} 
```
Note the `val` is **un**quoted.

5. Run it again after one pull (default 10min I think)
```bash
root@iobroker:/opt/iobroker# iobroker state get eufy-security.0.T8010NXXXXXXX.station.guard_mode                                       
{"val":"0","ack":true,"ts":1609259995023,"q":0,"from":"system.adapter.eufy-security.0","lc":1609259995023}
```
Note the `val` is quoted.

**Expected behavior**  
I would expect the type of `station.guard_mode` to be consistent. Either String or Number.

**Screenshots & Logfiles**  
```
root@iobroker:/opt/iobroker# iobroker state get eufy-security.0.T8010NXXXXXXX.station.guard_mode                                       
{"val":"1","ack":true,"ts":1609259693908,"q":0,"from":"system.adapter.eufy-security.0","lc":1609259693908}                                
              
```

**Versions:**  
 - Adapter version: 0.0.9
 - JS-Controller version: 3.1.6
 - Node version: 12.19.1
 - Operating system: Docker Container, Debian 10 Based https://hub.docker.com/r/buanet/iobroker/

**Comment by bropat** [comment]
@MaxWinterstein 
I can't reproduce your issue with latest version. The described issue was already fixed with version 0.0.9. Are you sure you were on version 0.0.9?

**Comment by MaxWinterstein** [comment]
Yes, seems like I mixed up two iobroker installations, d'oh! Sorry :(
